packages/javascript/plotlywidget/package-lock.json

```diff
@@ -9,26 +9,26 @@
       "resolved": "https://registry.npmjs.org/3d-view/-/3d-view-2.0.0.tgz",
```

Comment:
It shouldn't be necessary for this file to have changed unless you're changing the bundled version of plotly.js.  Could you revert it?

Reply:
This one will go away when rebasing on master. I don't think I changed the bundled version of plotly.js but I ran the codegen to update the docstrings.

Reply:
When you run codegen, are you using `python setup.py codegen`?  This shouldn't trigger any JavaScript/npm logic that would alter this file.  When you merge in master, go ahead and accept masters version of this file as-is.